common/app/model/dotcomrendering/DotcomRenderingUtils.scala

```diff
@@ -257,10 +257,7 @@ object DotcomRenderingUtils {
       if (firstEl.tagName == "p" && secondEl.tagName == "p" && secondEl.text().length >= 150) {
         AffiliateLinksCleaner.shouldAddAffiliateLinks(
           switchedOn = Switches.AffiliateLinks.isSwitchedOn,
-          section = content.metadata.sectionId,
           showAffiliateLinks = content.content.fields.showAffiliateLinks,
-          supportedSections = Configuration.affiliateLinks.affiliateLinkSections,
-          defaultOffTags = Configuration.affiliateLinks.defaultOffTags,
```

Comment:
Just confirming my understanding, uses similar logic as below to determine whether to add affiliate links?